libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/Expressions/ArrayExpression.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Converters;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// ArrayExpression - represents a property which is either a value of array of T or a string expression to bind to a array of T.
+    /// </summary>
+    /// <typeparam name="T">type of object in the array.</typeparam>
+    /// <remarks>String values are always be interpreted as an expression, whether it has '=' prefix or not.</remarks>
+    public class ArrayExpression<T> : ExpressionProperty<List<T>>
+    {
+        public ArrayExpression()
```

Comment:
>ArrayExpression [](start = 15, length = 15)

Looks like you are missing all of the comments on your constructors for all of your expression property things--have you been looking at warnings when compiling?